packages/lesswrong/server/database-import/database_import_new.ts

```diff
@@ -10,23 +10,23 @@ import mapValues from 'lodash/mapValues';
 import groupBy from 'lodash/groupBy';
 import pick from 'lodash/pick';
 import htmlToText from 'html-to-text';
-import cheerio from 'cheerio'
 import * as _ from 'underscore';
 import { Random } from 'meteor/random';
 
 const postgresImportDetails = {
   host: 'localhost',
   port: 5432,
-  database: 'oldforum',
-  user: 'jpaddison', // If this is the logged-in user on localhost, no need for password
-  password: ''
+  database: 'reddit',
```

Comment:
What's the story with this change? Is this just moving to more sensible defaults?